### DISCUSSION TITLE
Possible to override default resolver?

### DISCUSSION THREAD

**Discussion by chris-fibo** [mirumee/ariadne#933] [Q&A]
Is it possible to override the default resolver, or maybe use a fallback resolver that everything calls?

We use proto3 protobufs for our internal APIs. So the objects that the graphql is working with are these protobufs.
As I understand it, the default browser basically does something like this:
`getattr(obj, attr_name, None)`
Except proto3 scalars always exists, and when they don't they return a default value (0 or empty string) Which means the above code will never return None.
What I'd like to do instead is something along the lines of (or something similar)
```
if obj.HasField(attr_name):
    return getattr(obj, attr_name)
return None
```
Basically I need to check for the existence of the field first. 
I am hoping for some elegant way to do this, rather than create a resolver for every type.
